session.go

```diff
@@ -50,19 +53,19 @@ type Session struct {
 	client      *spanner.Client
 	adminClient *adminapi.DatabaseAdminClient
 	clientOpts  []option.ClientOption
+	priority    pb.RequestOptions_Priority
```

Comment:
The first time I read, I've confused with `tc.priority`.
I prefer more explicit name like `defaultPriority` for this field.

Reply:
Fixed.